src/Analyser.js

```diff
@@ -0,0 +1,50 @@
+// Given a list of snapshots, returns a list of snapshots that have expired
+// This can be determined using ExpiryDate tags
+let findDeadSnapshots = (snapshotList) => {
+	return [snapshotList];
+};
+
+// Helper function that checks if a single snapshot has expired or not.
+// Returns true if the snapshot has expired and needs to be deleted.
+let snapshotIsDead = (snapshot) => {
+	return snapshot;
+};
+
+// Given a list of EBS volumes and list of snapshots, matches each snapshot to the
+// EBS's Snapshots[BackupType] array. Returns an object containing the volume list
+// and a list of orphaned snapshots
```

Comment:
It could be a list of backups that are no longer necessary, or volumes that have vanished and may need urgent restoration.